nifi-nar-bundles/nifi-aws-bundle/nifi-aws-processors/src/main/java/org/apache/nifi/processors/aws/s3/ListS3.java

```diff
@@ -217,6 +218,15 @@ public class ListS3 extends AbstractS3Processor implements VerifiableProcessor {
             .defaultValue("0 sec")
             .build();
 
+    public static final PropertyDescriptor MAX_AGE = new Builder()
+            .name("max-age")
+            .displayName("Maximum Object Age")
+            .description("The maximum age that an S3 object must be in order to be considered; any object older than this amount of time (according to last modification date) will be ignored")
```

Comment:
@marcocharlie I had a look at the code and I think there is a typo here:
```suggestion
            .description("The maximum age that an S3 object can be in order to be considered; any object older than this amount of time (according to last modification date) will be ignored")
```
or `may be in order to be considered`.